File: PokemonShopping.Application/DTOs/ShoppingCartDTO.cs
namespace PokemonShopping.Domain.Models
{
    public class ShoppingCartDTO
    {
        public Guid Id { get; set; }
        public string TransferId { get; set; }
        public DateTime? PurchaseDate { get; set; }
        public decimal? Amount { get; set; }
        public List<ProductInCartDTO> Products { get; set; }
    }
}

File: UI/pokemonShopping/src/app/modules/api/models/shopping-cart-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ProductInCartDto } from './product-in-cart-dto';
export interface ShoppingCartDto {
  amount?: null | number;
  id?: string;
  products?: null | Array<ProductInCartDto>;
  purchaseDate?: null | string;
  transferId?: null | string;
}
